core/iwasm/libraries/debug-engine/handler.c

```diff
@@ -11,18 +11,29 @@
 #include "wasm_runtime.h"
 
 #define MAX_PACKET_SIZE (0x20000)
-static char tmpbuf[MAX_PACKET_SIZE];
+static char *tmpbuf;
 static korp_mutex tmpbuf_lock;
 
 int
 wasm_debug_handler_init()
 {
-    return os_mutex_init(&tmpbuf_lock);
+    tmpbuf = wasm_runtime_malloc(MAX_PACKET_SIZE);
+    if (tmpbuf == NULL) {
+        return BHT_ERROR;
+    }
+    int ret = os_mutex_init(&tmpbuf_lock);
```

Comment:
Had better define ret at the beginning of the function

Reply:
done